test/com/google/enterprise/adaptor/DocumentHandlerTest.java

```diff
@@ -1693,6 +1729,25 @@ public void testPercentEncoding() {
                  + "%2A%28%29%5B%5D%7B%7D%C3%AB%01", encoded);
   }
 
+  private static class SampleDocumentContentTransformer
```

Comment:
I added the testing class next to all other inner classes, so I thought it would be correct. Now I moved it next to the test function.